package-lock.json

```diff
@@ -1412,11 +1412,27 @@
       "integrity": "sha512-EUCPEkaRPvmHjWAAZkWMT7JDzpw7FKB00WTISaiXsbNOd5hCHg77XLA8sLYLFDo1zepYLo2w7GstN8YBqRXZfA==",
       "dev": true,
       "requires": {
-        "convert-source-map": "1.8.0",
-        "istanbul-lib-instrument": "4.0.3",
-        "loader-utils": "2.0.0",
-        "merge-source-map": "1.1.0",
-        "schema-utils": "2.7.1"
+        "convert-source-map": "^1.7.0",
+        "istanbul-lib-instrument": "^4.0.3",
+        "loader-utils": "^2.0.0",
+        "merge-source-map": "^1.1.0",
+        "schema-utils": "^2.7.0"
+      }
+    },
+    "@magnite/fledge.polyfill": {
```

Comment:
the vast majority of your changes to this file are not germane to this pr

Reply:
what do you mean?

Reply:
there are hundreds of changes to package.lock that are unrelated to this pr

Reply:
I installed a dependency and it looks like the `package-lock.json` updated mostly versions of babel processors

Reply:
I see you haven't fixed this; is your goal here to get a merge or something else? If you're not looking for a merge can you change to draft?

Reply:
the goal is to get this merged; you're reviewing the `package-lock` changes which are a result of a required dependency that was installed and affected the `package-lock`